baseframe/filters.py

```diff
@@ -178,36 +179,37 @@ def cdata(text):
     return Markup('<![CDATA[' + text.replace(']]>', ']]]]><![CDATA[>') + ']]>')
 
 
-@baseframe.app_template_filter('shortdate')
```

Comment:
You can't remove existing filters until these are _all_ gone. https://github.com/search?q=org%3Ahasgeek+shortdate&type=Code

Reply:
I'll perform a code review and phase out usage of existing filters.